webplugin/plugin.js

```diff
@@ -145,6 +145,12 @@ function createCustomClasses(classSettings) {
     document.getElementsByTagName('head')[0].appendChild(style);
 }
 
+// Set language and direction based on browser's language
+function languageDirectionChangeAuto() {
+    document.documentElement.lang = navigator.language.toLowerCase();
+    document.documentElement.dir = document.documentElement.lang === 'ar' || document.documentElement.lang === 'he' ? 'rtl' : 'ltr';
```

Comment:
@gauravbhatt007 You are modifying the page direction instead of the kommunicate iframe.

Reply:
sir i am scanning the the language and passing the direction to communicate iframe
![Screenshot (115)](https://github.com/Kommunicate-io/Kommunicate-Web-SDK/assets/122004908/48f42e7b-955c-42bf-83d4-78b15a2bb170)

Reply:
Done the required changes
![Screenshot (126)](https://github.com/Kommunicate-io/Kommunicate-Web-SDK/assets/122004908/e588414f-f69c-483a-bc46-eb249b743de3)